spec/spec_helper.rb

```diff
@@ -32,11 +61,11 @@ def grok_match(label, message, exact_match = false)
   end
 
   def build_grok(label, exact_match = false)
-    if exact_match
-      grok = LogStash::Filters::Grok.new("match" => ["message", "^%{#{label}}$"])
-    else
-      grok = LogStash::Filters::Grok.new("match" => ["message", "%{#{label}}"])
+    grok_opts = { "match" => [ "message", exact_match ? "^%{#{label}}$" : "%{#{label}}" ] }
+    if (compat = ecs_compatibility) != nil
```

Comment:
this setter in the `if` statement makes the code pretty hard to follow. Can we make it more readable?

Reply:
hopefully cleaner now, the idea was to leave the plugin ecs (legacy) default for existing specs